docs/madr/decisions/026-mesh-tcp-route.md

```diff
@@ -0,0 +1,144 @@
+# `MeshTCPRoute`
+
+* Status: accepted
+
+Technical Story: https://github.com/kumahq/kuma/issues/6343
+
+## Context and Problem Statement
+
+Now that we have the [new policy matching resources](./005-policy-matching.md),
+[new traffic route designed](https://github.com/kumahq/kuma/issues/4743), and
+[basic api of `MeshHTTPRoute`](https://github.com/kumahq/kuma/issues/5470)
+implemented, it's time to introduce `MeshTCPRoute`.
+
+### Goals
+
+Policy should allow to
```

Comment:
So does it apply to services marked as `http`?

Reply:
it does

Reply:
can you add it to the MADR?